custom_components/elmo_iess_alarm/const.py

```diff
@@ -5,6 +5,7 @@
     s.ELMO_E_CONNECT: "Elmo e-Connect",
     s.IESS_METRONET: "IESS Metronet",
 }
+CONF_ALIAS = "alias"
```

Comment:
This is a broader comment, so it applies to the entire PR. Instead of using `alias`, should we consider naming it `system_name` or `control_panel_name` (nome impianto/sistema)? While "alias" does make sense (as it is an actual alias), defining it as "nome impianto" might better guide users in deciding what to input there.

What do you think?